app/components/templates/templatesController.js

```diff
@@ -179,6 +179,8 @@ function ($scope, $q, $state, $filter, Config, Info, Container, ContainerHelper,
       var selectedTemplate = $scope.templates[id];
       $scope.state.selectedTemplate = selectedTemplate;
       $scope.formValues.ports = selectedTemplate.ports ? TemplateHelper.getPortBindings(selectedTemplate.ports) : [];
+      $location.hash('selectedTemplate');
+      $anchorScroll();
```

Comment:
It sort of works without location. When I remove location and rewrite the anchorScroll line it only works the 2 time I press any of the templates. The first time it move's partially to the ID but not the whole way. I figured it was do to the html ID not being able to be seen so I moved it out to an empty DIV. Maybe it would be better to move the ID up the "Application template list" or Templates